contributing.qmd

```diff
@@ -22,10 +22,251 @@ A file that has the name of your extension, the categories it falls into, and a
 {{< include _template/connect-extension.qmd >}}
 ```
 
-### `connect-extension.toml`
+# Connect Gallery
 
-A file with the name, title, a brief description of the extension, and the access type it should have on being created
+The Connect Gallery displays content that can be easily added to Posit Connect
```

Comment:
```suggestion
The Connect Gallery displays apps, dashboards, and reports that can be easily added to Posit Connect
```

Trying to see if using slightly more specific words than "content" here is helpful